pkg/hugo/frontmatter.go

```diff
@@ -134,5 +134,9 @@ func getAllStringItems(cfm *pageparser.ContentFrontMatter, fmKey string) ([]stri
 
 func getFirstStringItem(cfm *pageparser.ContentFrontMatter, fmKey string) (string, error) {
 	arr, err := getAllStringItems(cfm, fmKey)
+	if len(arr) == 0 {
```

Comment:
Thanks for finding the bug :)
getAllStringItems() check the array length inside, and then it returns FMNotExistError. In this case, arr is nil, so could you change it to check a err is not nil instead of length?

Reply:
I've fixed here as you said.